app/components-test-lib/page.tsx

```diff
@@ -67,7 +72,9 @@ const ComponentsTestLib = () => {
       <Text mb={6}>This section demonstrates Share menu component</Text>
       <VStack spacing={6} align="start">
         <HStack w="100%">
-          <Share />
+          <Suspense>
```

Comment:
Wrapping w/`<Suspense>` appears to be the correct for the prerender errors we were seeing on eg Next 14.